packages/selleo-design-core/src/Avatars.tsx

```diff
@@ -0,0 +1,74 @@
+import { h } from "preact";
+
+import cx from "classnames";
+
+export type AvatarProps = {
+  hideName?: boolean;
+  imageSize?: number;
+  image?: h.JSX.Element | null;
+  username?: string;
+};
+
+export function SmallAvatar({
```

Comment:
Please create one avatar component that will take size = small | regular

Reply:
Okay, I've decided to name size "regular" | "big" since it'll help me when I'm passing a prop to a Testimonial and finally to the Avatar (to match the names)

Reply:
they are not  labeled in figma so I think it is okay :D
 If on the other hand it would have labels on figma let's stick to it 😄

Reply:
Yes, good point ;)